From 71893520532770d43cca70f1e2c430e568efe53f Mon Sep 17 00:00:00 2001
From: Richard Danter <radanter@gmail.com>
Date: Mon, 20 Jan 2020 09:28:11 +0000
Subject: [PATCH] Fix compile error caused by incorrect include path

The Makefile set the include path for the staging directory relative to
the current directory rather than the kernel source root.

Signed-off-by: Richard Danter <radanter@gmail.com>
---
 drivers/char/broadcom/vc_sm/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/char/broadcom/vc_sm/Makefile b/drivers/char/broadcom/vc_sm/Makefile
index 20b7646fe4b6..6bde1b91594a 100644
--- a/drivers/char/broadcom/vc_sm/Makefile
+++ b/drivers/char/broadcom/vc_sm/Makefile
@@ -1,5 +1,5 @@
 ccflags-$(CONFIG_BCM_VC_SM) += -Werror -Wall -Wstrict-prototypes -Wno-trigraphs -O2
-ccflags-$(CONFIG_BCM_VC_SM) += -I"drivers/staging/vc04_services" -I"drivers/staging/vc04_services/interface/vchi" -I"drivers/staging/vc04_services/interface/vchiq_arm" -I"fs"
+ccflags-$(CONFIG_BCM_VC_SM) += -I"$(srctree)/drivers/staging/vc04_services" -I"$(srctree)/drivers/staging/vc04_services/interface/vchi" -I"$(srctree)/drivers/staging/vc04_services/interface/vchiq_arm" -I"fs"
 ccflags-$(CONFIG_BCM_VC_SM) += -DOS_ASSERT_FAILURE -D__STDC_VERSION=199901L -D__STDC_VERSION__=199901L -D__VCCOREVER__=0 -D__KERNEL__ -D__linux__
 
 obj-$(CONFIG_BCM_VC_SM) := vc-sm.o
-- 
2.23.0

